apps/console/src/features/organization-discovery/components/add-organization-discovery-domains.tsx

```diff
@@ -273,17 +292,19 @@ const AddOrganizationDiscoveryDomains: FunctionComponent<AddOrganizationDiscover
                             </FormHelperText>
                             <FormSpy subscription={ { values: true } }>
                                 { ({ values }: { values: AddOrganizationDiscoveryDomainsFormValuesInterface }) => (
-                                    <PrimaryButton
-                                        data-componentid={ `${componentId}-form-submit-button` }
-                                        disabled={
-                                            submitting || isEmpty(emailDomains) || isEmpty(values?.organizationName)
-                                        }
-                                        loading={ submitting }
-                                        type="submit"
-                                        style={ { marginTop: "20px" } }
-                                    >
-                                        { t("common:update") }
-                                    </PrimaryButton>
+                                    !isReadOnly && !hasScopes && (
+                                        <PrimaryButton
+                                            data-componentid={ `${componentId}-form-submit-button` }
+                                            disabled={
+                                                submitting || isEmpty(emailDomains) || isEmpty(values?.organizationName)
+                                            }
+                                            loading={ submitting }
+                                            type="submit"
+                                            style={ { marginTop: "20px" } }
```

Comment:
Shall we avoid adding inline styles

Reply:
Add this to `add-organization-discovery-domains.scss`.

Reply:
Fixed with f14b8e1c35cbf764611d52797b803c6294e0fc0f